contents/docs/surveys/implementing-custom-surveys.mdx

```diff
@@ -83,4 +83,50 @@ posthog.capture("survey sent", {
     $survey_id: {survey.id} // required
     $survey_response: {survey_response} // required
 })
-```
\ No newline at end of file
+```
+
+### Question types
+
+If you want to make the most of PostHog's automatic survey analysis, you'll need to capture survey responses in these formats:
+
+#### Open text and single choice questions
+
+```js-web
+posthog.capture("survey sent", {
+    $survey_id: {survey.id} // required
+    $survey_response: {response} // value of the text input or selected choice
+})
+```
+
+#### Multiple choice questions
+
+```js-web
+posthog.capture("survey sent", {
+    $survey_id: {survey.id} // required
+    $survey_response: ["response_1", 8, "response_2"] // array of values of the selected choices
+})
+```
+
+#### Rating questions
+
+```js-web
+posthog.capture("survey sent", {
+    $survey_id: {survey.id} // required
+    $survey_response: {rating} // rating is a number
+})
+```
+
+## Multiple question surveys
```

Comment:
This is a bit of curveball, since so far in the docs we havent mentioned multiple question surveys. How do you create these in the PostHog UI?

<img width="1305" alt="Screenshot 2023-10-09 at 9 39 43 AM" src="https://github.com/PostHog/posthog.com/assets/7090054/d4057dcf-9e25-4a0f-8de5-b035e88eae11">

Reply:
Multiple questions is released after the beta period ~ but it'll be a "paid for" feature

Right now it's under a feature flag